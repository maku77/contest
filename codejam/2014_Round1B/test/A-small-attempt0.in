100
2
pmwpshndsckzghhixvrcpncazxojruoiikyauulfopqnyzxgzsybjnflncbyuvmefzcgvqljhdz
pmwpshndsckzghhixvrcpncazxojruoiikyaulfopqnyzxgzsybjnflncbyuvmefzcgvqljhdz
2
r
r
2
gimcbuzxamfctkdycyuraeezybqaqwjwgxaitbhwnnyixdibbdsehzdhcvvhstsqqas
giiimcbuzxxamffctkkdyccyyyuuraeeezybqaaqwjwgxaaitbbhwnnnnyixxdibbdsssseehhzdddhhhhcvhsstsqas
2
egcaeougkijxzvrtryoojldbgbahmniqumtabpgnxpnwnergchvouarabtkqguiajbaksjzpyonntguxnpnjr
egcaeougkijxzvrtryoojldbgbahmniqumtabpgnxpnwnergchvouarabtkqguiajbaksjzpyonntguxnpnjr
2
wxxrrrrrrrrrrrpbbpbbbbbliffxxxiikkhhqqhhhhhhhhhhqqwkkkkkkkkkkkkkkkkkkkdlllllloyu
wwwwwwxxxxxxxxrrrrrrpbbbbbpppbblifxxxxxxxxikhhhhhhhqqqqqqqhhhhhhhhhqwwkddlllllooyyuuu
2
pcbhproqojdpayqvmbftejulcdflwyvchjpdbyspjbbtucklmpqzidnoeelwawmqkkwmrhltnujvzq
csyoqejqcduumabcpemlbbpymzafhbpphwlvjfoyqtjjmluqwrbipnocrpdkkhvwkvtwqtjnldedzl
2
gggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggggb
gbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbbb
2
czpainhfbqpggr
czpinhfbqpggr
2
hyrkhiuwlmncejpypxwwwaatfgoynrracccjbjhuvpesnpppppgarr
hyyyyrkhhiiiiuuwwlllmmncccccccejpyyypxxwwaatfggooynraccccccjjbbbjhhuuuuvvppeessnnpgar
2
bfkbnqjezzbjrydqatxgzfcgputfkunscxvpnevodwzuxdmxykfxphfgezlovyjxxeokljaoialfdxebjkyyrdevcrmyrxxodoa
fkbnqjezzbjrydqatxgzfcgputfkunscxvpnevodwzuxdmxykfxphfgezlovyjxxeokljaoialfdxebjkyyrdevcrmyrxxodoa
2
lgfhokysvbuxktawmalpmsuugtvlxprt
zgfhokysvbuxktawmalpmsuugtvlxprt
2
bmugzczitinutbxpmwsdmgqfnlfcceznhnjnnlpvgzlcsmzhzyaemgwtvjyy
bbmugzczzzitinuutbbxppmmwsdmgqfnnnlllfccezzzznhhnjnnlpvggzzlllccsmzhzyyyaemmmmgwttvjjy
2
ssffnnhhhhhassshhhhhhhhhhhhhhhptttttoossssssssssscccccccccccccceeeeeeeeeeeeeeeeeeeee
ssfffffffffffnhaaaaaaaassssssssshppttttttoooooooooooooooooooosssccccccccccccccccceee
2
nelyjdevbobelcyzuxtmrfsvwdqhrvwtjjhvqbipdatfqbrxczlvhhitjqnluggphlvuouggrxmkzmphp
xzmudxmrvwgladmlyeyqhgpqdbjcvbntohriqupigwjehpuhntlbhovjcvtzvpbhkqftgvzfljsuxlerr
2
vnoccdzmmalbytjzekmrwabfffqngftpkggnzurrrevvdhihfdqgkdsbbbzuuegaxxuiccnaaownpsseevvq
vnocdzmalbytjzekmrwabfffqngffftpkkgnzurevdhihfdqgkdsbzuueegaxxuuiccnaownnnpsevvqq
2
pyqlgbssgnxtjrcxhszztukozneabblqccdiifvdolajwtnnvafuabqkpqqavqbzbtbfwlntobeqmclzymeuohlawbautdtwwue
pyqlgbssgnxtjrcxhszztukozneabblqccdifvdolajwtnvafuabqkpqavqbzbtbfwlntobeqmclzymeuohlawbautdtwwue
2
ufcezhiztsxzyezlshzmpbgauzgdvterzgxynizgawgadhmxqnjgprglroqoiufhcehpmixpgfpjobgfrslijtvchlqrhxajch
ufcezhiztsxzyezlshzmpbgauzgdvteerzgxynizgawgadhmxqnjgprglroqoiufhcehpmixpgfpjjobgfrslijtvchlqrhxajch
2
dhl
bdl
2
occccccyoouuuuuuuyyyyyyykkkkkkkkkkkkkkkkkkkkkmmmmmmmmmmisssssssssssssugggggggggggggggggggggg
ooooooocccccccyyyyoouuuuuuyyykkmmmmmmmiisssssssuuuug
2
goyctetyntyneujkxlkrzwbdabgywbecpcgkgbjvuh
hxjnsmtkcefkdivofrwciakxdqoqdnjkktxehsolzvvcfrslkppspzruqfktuwferfiyxyjwwgbbztmkidezdwtt
2
riqqcgghxdnttiittsssssssmmcddwttwmmmwwkasssssjxnnhhsshmllrrrrrrrcnyllnnnbbbziil
rriqqqcggghxdntitssssmcccdwttwwmmmwkasjjjjjxxnnhshmmllrrrrrrrcnyylllnnnnnbbbzziiiiilll
2
fkkprocsllckhznjtsmvohtvgvacesmmhhjzrzegcpzzuuhbmoxguvueqjbtdticcpasmxmoeqfjummbsspmbewmxlyxngvos
szxuqzhcjmoecvhzfpnttjesmcrrmompusbgkuavulcoptsocmvxoekghlvzvgsbhmpxxdyukzlmebjtwnqsgaeimbmmjsfhc
2
vbfhkfyzxkbwudcntl
vbfhkfyzxkbwudcntlc
2
lkaysspuommanlopgmopvtvietuyvwwhgwfbquveikgwxunggbxewso
akaysspuommanlopgmopvtvietuyvwwhgwfbquveikgwxunggbxewso
2
ggkkkkkkppwfllllllzzhhhqhhhccccfdddddffqqqpdwhhvzyyixhiyyaqqjjggbyyyyyyyyyyeegggjjfnnnbnwwwwwffsaal
ggggkppwwfllzhqhhhcffddfqpdwhhvvzyyiixhhiiyaaqjgbyyyyyyeeeeeeeegggjfnnbbnwfsall
2
neqhhqvzgjdajknaazugahggthudtaszppimtaulytthyutihthebkliffuueknecmvmzzqptcnoxwntpxbkh
neqhhqqvzgjdaajknaazugahhggthudtasszpimmtaaaulyyytthyuutihthebklifueeknnecmvmzqptcnoxwntpxbkhh
2
zzzbxxupoffffffxxuttvvnnvvvvvcggggkklllnnnnnnnnccccc
zbxuuuupppoooooofxxxuuttvvvvvvvvvvvvnvvvvccgggggggggggkllllllnnnccccccccc
2
yxhgfunnwgpsmrexrflincnbkhdvvejwdseimryizpalghlymyibavcmfhicltaolezxxzhxojkwqvweuffwcjjjqtodmouxvt
yxhgfunnwgpsmrexrflincnbkhdvvejwdseimryizpalghlymyibavcmfhicltaolezxxzhxojkwqvweuffwcjjjqtodmouxvt
2
zzzzaaaaaaaaaaaaaaa
zzzzzzzzzzzzzzzzzzzzzzzzzzzzzaaaaaaaaaaaaaaaaaa
2
efrhpaznnihewrbiqjvxfufrpqkfzubfzsnqu
efrhpaznnihewrbiqjvxfufrpqkfzubfzsnqum
2
dodbxxxhhblydnuzbghkbxrqobagtuufpzztsbsricvvaylwxmcgyfffryvtydmsidsevvynfakadinzibeq
dodbxxhhblyyydnuzzbghhkbbxrqqobbaggtufpztsbbsriccvvaylwxmcggyffryvtydmsidsevvynnfakkaadinzibeq
2
nekyuzkkmmpmstxltewyhterrymgmmplhajjxgguxcvjjokxjggwubbaqziijffffbkrhww
nekyuzkmmpmsttxltewwwyhterymggmplhajxguxxcvjokxjgwubaqziijfbkrhwww
2
rrrrrrrrrrrvvvvvvvggggggggggggvvvvvvvvvvvvvvvvvvvvvvvvvvbbbb
rrrrrrrrrrrrrrrrrrrvvvgvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvvb
2
spxeppdofgcivajahmsuuhnbgoigkumcljicalriutqrvbtfplbjsokbeujoqvtcgbghpxpmjthaqitvxg
spxeppdofgcivajahmsuuhnbgoigkumcljicalriutqrvbtfplbjsokbeujoqvtcgbghpxpjthaqitvxg
2
pppkkkkxdaaaapllccbqqrrrryyvfssnnnjjjjjallhhhqeeeeeeggjjjvv
pppppkxxxxxxxdddaappplcbbbqqryvvfffsnnjjjjjallhqqqqegjjjjjv
2
gtajdnriaenzluxmklysksuoxmhdbcvojoqstdlcfppec
gtajdnriaenzluxmklysqksuxmhdbcvojoqstdlcfppec
2
g
azbuibhdoxqqrwtopiuanjjlfglzesqgttddxkgliydb
2
wnpusqkpnpxzxvzuifalcgjuumieoonnechyssphjnhgigatndgpjqlgevmsmzhqbpqvjgdsvmyfubyhehypxlxh
wnpusqkpnpxzxvzuifalcgjuumieoonnechyssphjnhgigatndgpjqlgevmsmzhqbpqvjgdsvmyfubyhehypxlxv
2
uujjjmfgtqmsswcllfiattxkkwzwxriaohkjqtudycwoorrhhkturdukbggiaeguumiitstmlnp
ujjjjjjmfgtqmssswclfiatxkwzzwwxxriaaohkjqtudycwoorhhkkkturdukbgiaegumiiitsstmlnp
2
yzrvlugooswwqdrkaooziserhhstuucxycyqylmezgcpjwzzlkommmuisnraikueprdrrvppchvvbnyqywssjiiwcqqsi
yzrvlugoswqdrkaooziserhhstuucxyccyqylmezgcpjwzlkomuisnraikueprdrrvpchvbnyqywsjiiwcqqssi
2
viomnzonshgzksgvwagqlzkbapfhbyoygcntbbjuipttjzqhbwanykoybthftvfazsvdvgxexszgtpquoqqjmcxmgtvl
viomnzonshgzksgvwagqlzkbapfhbyoygcntbbjuipttjzqhbwanykoybthftvfazsvdvgxexszgtpquoqjmcxmgtvl
2
wffgelzzzypscccttbbdjjboxttczqdjjrzvvsruuhosnuuuwyyyv
wfggelzzzzzyyppppppsssccctbbbdjjboxxxtccczqdjrzvvvssrruuuuhosnuwyyvv
2
ddgyxicckkkggggggsssslyyiccccczzwzzzbbtklrvvvvvvbbbotttttttkktttyynnnnnnweoovn
ddddgyxxicckkggssssllllyicczzwzzzzzzbtkllrvvvvbbbbbooooootkkttyynnweeoovn
2
luyiieyuzdbprycydwqtpdyamydrxveeiiresziswjxxyyowdobzexqixrwxxwnuttelxykhqgqngrdewrfvwe
luuyiieyuzdbprycyydwqttpdyamydrxveireszisswjjxxyowdobzexqixrwxwnnuttttelxyyyykhqgqqngrrdewrfvwwe
2
mucwxlfiqhhbgbpqpyzerwaferrjoxtrbfqjhrvrzhxmmycgorifpbpudotjgtrwadwlwwpithzoaumjcjesthhjvxzioei
btjrghvaxjytphfhsducjjthfmlrgrriezrppzaeyipqxvcflzbwrbohbwwxjwtofoqcuhxwhwuigeiorpmditjmqozmaer
2
eessssssssyyyyyyyygggggggggggffffffffffdddddiuuuuyyrreeeeeeeeeffffdddddbbbbttttttbbbbbjj
essyygfdiuyrefdbbtbbjj
2
wleqvgzxjlpsnhsbwgwdauqrjckomfliqraoyalhncbaltcizylbrudueffutkfdd
wleqvgzxjlpsnhsbwgwdauqrjckomfliqraoyalhncbaltcizylbudueffeutkfdd
2
ecdxnuhetyctuadkyobqqoeosvadtjkylnv
ecdxnuhetycteuadkyobqqoeosvadtjkylnv
2
uzbasdrhgabcdqkaskrbdrsjxvvsciqwktyeyplgrpivhsvzfoaiitthqrzsbrqls
uzbasdrhgabcdqkaskrbdrsjxvvsciqwktyeyplgrpivhsvzfoaiitthqrzsbrql
2
kjswlgrpmkvmzhlfuyijtltqg
kjswlgrpmkvmzhlfuyijtltq
2
qmfzbmnqoooooqkkdpretxxzjlliitqjyttfmzvaggxcskujyylvkqqq
qmfzbbmnqooqkkdpretxzjlitqjytfmzvagxcskujjyllvkqq
2
qomhehaiiiodddrxsooddjvffcfaazzcgyykkhmekfuuzzzmaiolwrccvvvmsjjjjjppegxxglxxfppefffbnsshhhzxxxbylch
qomheehaiodrxsoodjvfcfazcgykhmeeekfuzzmaiolwrcvmsjpegxglxfppefbnshzxbylch
2
kwsenijwjdvwhbwhcxkcyjusidwhhyctuwyhfhfokdmtekdihnkiwgahjzqrxumtrkbwugmglyapkdx
cvtldzvwdjwjzcktqkskoomiejwvzyxbvqpbrmxuvvgvysqofjytxnedyabzbybyqszhexez
2
ojjjjdjjuuugxbvibnnnnsxxhwwwuvwwsjjmffqupffvcnkkryoocswztrrrhwweebbvnnyypiwhukpmgercigmzeccb
ojdjugxbvibbbnssxhwuuvwssjmfqupfvcnkryocswztrhwebvnypiwhukpmgercigmzecb
2
zzxlcylckkkuoozzxttxbooaiiiaaalobbshyyeelwamvzxxxzickeqqjcjidzzklawzbojbmpmkpyyiipvvfogjakzkmkuurrrm
zzxlccylckuozxtxxboaialllobshyyelwwamvzxxxxzzziickeqjjcjidzzklawzzbojbmpmkppyipvfogjakzkmkurrm
2
lrgoypmjbppcyuwwttjrlvjkcjtvzabwyonecnuryvgavcyoxhijertgaodbqgzoovmtzzpqamyckdbaakijpb
lrgoypmjbpcyuwtttjrrlvjjkccjtvzabwyonnecnnuryvgavvcyoxhiijerttgaodbqqgzovmtzzpqamyckdbbaakijpb
2
qvqntxiwvigsylznokcdwbsyfesoxajpwcdpznnuxvoxiqkxapayqsyxxrlwrwlpyogzdtvdrmazenwgcweurdtqwgmpcxfdon
qvqntxiwvigsylznokcdwbsyfesoxajpwcdpznnuxvoxiqkxapayyqsyxxrlwrwlpyogzdtvdrmazenwgcweurdtqwgmpcxfdon
2
fvhhhvwwqapilzrguuqqhhmhommixorzzejzkhhgfqqbdgcvopuiypeowqvllcfkssxmzggljppttqfutmyocqjkoaooccww
fvvhvwwqapilllzrguuqhmmhooomiixorzzejzkhgfqbbdddgcvopuiypeowqqvlcfksxmzgljjpptqfutmyocqqqqjjkoaaocww
2
yykjglczvpdjppbeohbsjyyxhmydtayrajciugipwlbldcsrltllwtludysituuhmusppeovmggtahfdlzwxuhvotpttjbmf
yykjglczvvppdjppbeohbbsjyyxhmyddtayrajciiugipwlbldcsrltlwtluudysituhmmuspeovmmgtahfdlzwxuhvotpttjbmf
2
f
ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff
2
zzbpeeyyjweqqpqqemdhrfzyqffucayjfuuiimtlbajmmypdpijblssusttiyyebbba
zbbppeeyyjwweqqqpqqemmmmdhhhhhhhrfzyqffffffuuuucayyyyjfuimtttlbbajmypdpijjbllllsustttiiyyyeba
2
pmzebnakonftfkuagisklvizpqvdddrasqgtfgetwlodvjddtxngtwfkodnrigtaxbuckayglmjjxpmsmcyhaetqjhjrpcso
omzebnakonftfkuagisklvizpqvdddrasqgtfgetwlodvjddtxngtwfkodnrigtaxbuckayglmjjxpmsmcyhaetqjhjrpcso
2
drmhnjubozaahzffmdousntxhpmsahfjz
srmhnjubozaahzffmdousntxhpmsahfjz
2
iskkkksssszzzrjjjjjjjjjffffffpzzzfczaaaaaaaaaaaauuunnqqqmmmmmmaai
isskkszrjjfffpzfczauunqmaaaai
2
hhhkbbbbbbbbbbbtttteeeewwwwwwwwwwwwwvvvvvvvvzzzzzzzzzzmmmmmmmmmmmmmgqqqqqqqqqqqqqqqqcccccccca
hhhhhhhhhkkkbttttttteeewwvzzzzzzzzzzzmmmgggggqqqqqqqqqqqqqqcccccccccccca
2
ebbbbbvvvvvvvvvvvvkkkkkkkvvvvvvvvvvvvvwwwwwwwkkeeeeeeeccccccccccccrrrrrrrr
eeeebvkvvwwwkkkkkkeccccrr
2
lovguuumsmlahdscrinfnnrtgmewwwfwnahbvoxsargosotmsmvfrkskdawjxehmeppbfmtheaaxyqqqkkichpvvriffwkke
lovguumsmllahdscrinfnnrrtgmewwfwnahbvoxsargosootmsmvfrkskdawjjxehmeppbfmtheeaxyqqkkichppvriffwke
2
tuquhyjaqebrwcqcytohoyeddttucqqjgjf
tuquhyyyyjaaaaaaaaaqebbrrwwcqqqqcyttttttttttooohhhhooooyyeeeedtuucqqqqqjggggjff
2
jovfnhghsyawjwqldgfsvtxvbhtefwnokjuasahlajjlfawkieddxcyzjtfopsdcdzewznjbws
dsloaptfm
2
aaooozzzzzzuuuumyyaaaaaqwwwwweeekknnnjjjddddffffffff
aaaaooozzzzzzzzzuuuuuuummmyaaqqwwwweeeekkknjjjdddddfff
2
ssmvwzqjjyuuuokkaomoodirfffurlobbbcnnngtmymmemjdddddoocm
ssmvwzqqjyyyyuokaomoodirfuuuuurrloobcngttmyyyymmeemjddddocm
2
sqnebiejqpuvmqqcdbsasbkxqnvadptkpeddcysexehjzggpwewapxzbwbbtledyohtjacvczlztmzioewkpe
mzbopuayceeqxxpqdlkpzsjcotwdavpabybptebsjhbwepzeatxqinwgnbsckdtwvejdkeicqhmqgsvedzzel
2
rrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrr
rrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrrr
2
ayuqspwbxwcsdnbqiohyvgsnsnkxynrhylszsfdw
iabrdxywssnfbznyvhscwgqukplqdssnnxhwoysy
2
iiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiihhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhhjjjjjjjjjj
iiiiiihhhhhhhhhhhhhjj
2
ovqjfmzydstpthuehygbezqayyxgqwyeuqpbcqbikuxgbtkkrslvubyucvbstzypppqquirctnbzoul
ovqqjfmzydsttpthuehyggbbeezqqayyxgqwyeuuqpbcqbikuuxggbbtkkrssllvubbyyyuucvbstzyyppppqquirctnbzoooul
2
xnjmmptotjjjpppehiobbbwwyzwfdttttggkffxhpzzggfkkswwdgrppwvvwejxdhhxcgfhhzzyynlddmmssuuoy
xxnjjmptoooootjpppeeeehhhiobwwwwyzzzwfdttggkkfxhppzgfkkswwdgrpwvwejxxxdhhxxxcgfhzynlllldmsuoy
2
tnsuxtbcfgtayivxrkjlmrrxsnsgqexjurfrlgvspqunzpnqzweopxnhkfpamplgirxvzuooi
tnsuxtbcfgtayivxrkjlmrrxsnsgqexjurfrlgvspqunzpnqzweopxnhkfpamplgirxvzuoom
2
ytaavcmfisssjxsyzdeehhgdiaogkcfxngglbhhgtoonngxxkaxjdbslfclvlwxqtkyzxcieidvwqdvcgsoltgxalkxwgu
ytavcmfissjxsyzdehgdiaogkcfxngglbhhgtoongxkaxjjdbslfclvlwxqtkyzxcieidvwqdvcgsoltgxalkxwgu
2
condfheessetfnwttlpxdjkkukmavxqcakfoakwhqolljvaiqnxnqjacfkphggoxxiojpejbaatrqedmhlcu
condfhesetfnwtlpxdjkkukmavxqcakfoakwhqqoljvaiqnxnqjacfkpphgoxiojpejbatrqedmhlcu
2
whqxmwhduvqfxgeiwlyslxhxcxmyotrmdjmpgttcplhor
ywhqxmwhduvqfxgeiwlyslxhxcxmyotrmdjmpgttcplhor
2
ggggzzzzzzzzkkcmmmmmxxxxzzzvooiprrrzzzzzmmvvvvhggggobbcccvvvvvbbbbner
gggggggzkkkcmmmmmmmmmxxxzvvvvooooooiipprrzzzzzzzmmmmvhhhhhhhhhhhhhhgoobbbcvvvvvvvbbbneeeeeeerr
2
aycbuafawxereequqqydkviarfuovthvtmyomdokdvchzsdrkeuuzewqjqfekocfactmikxlhb
uvcewsethxynmyeltufquvsxqz
2
iihhhwwwwccyeccyyrrrrrrffvwazzefft
iiiiihwwwwccccyyyyyyyyyyyecccccyrfvwwazzeefftt
2
inkkytttppithhhthhhhcssssssgkkhhffffffffgacjrjzzzzzooooouuuuuuuuqqbttzjjjjmmhc
inkytpithtthcsgkhfgacjrjzooouqbtzjjmhc
2
z
zz
2
vlrubfhqiecuxbyegfaapxgmpooomwppjxlxpfawoqguvcrwaveixvfdjwtyirpupcrgktcyjitek
vlrubfhqiecuxbyegfapxggmpoommwpjxlxppfawoqguvcrwaavvveixvfddjjjjwttyirpupcrgkktcyjittek
2
whjuqjwpvwdqnstwejezehqpqfjgdygciqyabwqwsvofnidttitxrlojqzpwawyiowkpsanmwbrl
whjuqjwpvwdqnstwejezehqpqfjgdygciqyabwqwsvofnidttitxrlojqzpwawyiowkpsanmwbrlw
2
yalmtpdcmye
yalmtdcqmye
2
bqmgprhzu
bqmgprhzh
2
kkkkkkkkeeeeeeeeeeeeeeeeeeeeeeyyyyyyyyuuuuuuuuuuuuuuuuaaaaooooo
kkkeeeeeeeeeeyyyuuaaao
2
sante
santt
2
wzvugguyiqblpmjoqkoqmzggozqbkfggbvveaszhpzrfadwwomlggwbxkbdbatckbknyppqqqqzhrqmttjiiuo
wzzvugguuyyyiqblpmjoooqkoqmzzggozzqbkfffgbveaszzhppzzrfadwomlgwbxkbdbbbatckbbknyypqqqqzhrqmttjiiuo
2
zzpkekapxldycpciadvdrrthohtffkdgdkhsuncwebbkhbozewbjtedoltwfaikilwqfsdjjppinbropqunnrgiwl
zpkekapxldycpciadvdrthohtfkdgdkhsuncwebkhbozewbjtedoltwfaikilwqfsdjpinbropqunrgiwl
2
ydgnxtqmmklykfjkf
fdvnsnqdxhqvmobudyntmbfxbpdlbixjnswggmlftbdhreduespquuovltinbfwpasxgginblqlcwoyah
2
iqcszkodaiejaazxllyyyzzsjjnfcvzcvvsaaodrrrvvsddjjjvfuboowcooiiomjixxobupf
iiqcsszzkodaiejjazxllyyyzzssjnfcvzzzcvsaodrrvsdjvfubowccoiomjixxobbuppf
2
qprnfrfukdtfhrarukmubxvfaqqburvagopnhxhuadziuzbpmnlnkiunznqvelwncnajkidnlfxhey
kprnfrfukdtfhrarukmubxvfaqqburvagopnhxhuadziuzbpmnlnkiunznqvelwncnajkidnlfxhey
2
oggggxaafxxoollllrrrccccvvveettfffffffffffnaaaiiiiiiiiiixxhedddbb
ogggggxxxxxxxaffffxolllrrrrrrrrrcccveeeettttfffnnnnnnnnnaaiiiiiixxxhheeeeeeedb
2
gmqtlpceswvyknwwjhrufvvpdyycctttwwwihfvayippanhyihkxehinaacuvtxtrbookvthhhvrpnucwwwwsmefllkkk
ggmqtlpceswvyykknwwjhrufvpdyyctwihfvvayipanhyihhkxehinacuuvvtxttrboookvthvrpnucwsmeflk
2
oniinnmmmmypvvtvvnnvjnugbjddddddmvvvvgggggookktbbbbbs
oninnmmmypvvtvnvjnugbbjdmvvvgggoktbbss
