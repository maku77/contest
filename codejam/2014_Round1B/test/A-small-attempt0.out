Case #1: 1
Case #2: 0
Case #3: 29
Case #4: 0
Case #5: 69
Case #6: Fegla Won
Case #7: 196
Case #8: Fegla Won
Case #9: 45
Case #10: Fegla Won
Case #11: Fegla Won
Case #12: 28
Case #13: 90
Case #14: Fegla Won
Case #15: 19
Case #16: 3
Case #17: 2
Case #18: Fegla Won
Case #19: 68
Case #20: Fegla Won
Case #21: 37
Case #22: Fegla Won
Case #23: Fegla Won
Case #24: Fegla Won
Case #25: 54
Case #26: 17
Case #27: 53
Case #28: 0
Case #29: 28
Case #30: Fegla Won
Case #31: 18
Case #32: 17
Case #33: 55
Case #34: Fegla Won
Case #35: 44
Case #36: Fegla Won
Case #37: Fegla Won
Case #38: Fegla Won
Case #39: 19
Case #40: 10
Case #41: 1
Case #42: 29
Case #43: 41
Case #44: 18
Case #45: Fegla Won
Case #46: 66
Case #47: Fegla Won
Case #48: Fegla Won
Case #49: Fegla Won
Case #50: Fegla Won
Case #51: 13
Case #52: 32
Case #53: Fegla Won
Case #54: 31
Case #55: 30
Case #56: 14
Case #57: 1
Case #58: 30
Case #59: 12
Case #60: 99
Case #61: 40
Case #62: Fegla Won
Case #63: Fegla Won
Case #64: 42
Case #65: 61
Case #66: 63
Case #67: 12
Case #68: 48
Case #69: Fegla Won
Case #70: 26
Case #71: 27
Case #72: Fegla Won
Case #73: 0
Case #74: Fegla Won
Case #75: 66
Case #76: 20
Case #77: 43
Case #78: Fegla Won
Case #79: 7
Case #80: 9
Case #81: Fegla Won
Case #82: 65
Case #83: Fegla Won
Case #84: 30
Case #85: 42
Case #86: 1
Case #87: 16
Case #88: Fegla Won
Case #89: Fegla Won
Case #90: Fegla Won
Case #91: 41
Case #92: Fegla Won
Case #93: 22
Case #94: 7
Case #95: Fegla Won
Case #96: 22
Case #97: Fegla Won
Case #98: 59
Case #99: 25
Case #100: 19
